src/components/ValidationMessage/ErrorValidation.tsx

```diff
@@ -0,0 +1,25 @@
+import ErrorIcon from './ErrorIcon'
```

Comment:
Naming is fine, but I got a bit confused. It's really subjective, but I would consider renaming this component to `ErrorPopup` or `ErrorMessage`. The current name makes me think that the component is validating something where in reality its just displaying a validation error message.